### PR TITLE
Create frontend table api data

### LINKED ISSUE #5: Create Frontend table API data

### PR DESCRIPTION
**Pull request by codyrhoten**
Pulled some data from the API and styled the table

### CODE REVIEW

File: components/ContractDataTable.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <div className={classes.table}>
+            <h2
+            /* size="xl" style={{ color: theme.colors.dark }} */
+            >
+                Contracts and Tokens
+            </h2>
+            <Table>
+                <thead>
+                    <tr>
+                        <th style={{textAlign: "center"}}>Contract Address</th>
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: Trim the Contract Address ex: 0x123...0U7J.
When you hover over the trimmed address it should show the full address use a mantine tool tip

File: components/ContractDataTable.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                        <th style={{textAlign: "center"}}>Name</th>
+                        <th style={{textAlign: "center"}}>Symbol</th>
+                        <th style={{textAlign: "center"}}>Total Balance</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {contractAddresses.map((contractData, i) => (
+                        <tr key={i}>
+                            <td>{contractData.address}</td>
+                            <td>{contractData.contractDeployer}</td>
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: Trim the contractDeployer Address ex: 0x123...0U7J.
When you hover over the trimmed address it should show the full address use a mantine tool tip

File: components/ContractDataTable.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        textAlign: "center"
+    },
+}));
+
+export default function ContractDataTable({ contractAddresses }: ContractDataTableProps) {
+    // const theme = useMantineTheme();
+    const { classes } = useStyles();
+
+    return (
+        <div className={classes.table}>
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: Place inside a Mantine container. It will give proper margins

File: components/ContractDataTable.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            <h2
+            /* size="xl" style={{ color: theme.colors.dark }} */
+            >
+                Contracts and Tokens
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: Replace `Contracts and Tokens` with `Deployed Contracts by <wallet address>. Put the wallet address on a second line.

File: components/ContractDataTable.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            </h2>
+            <Table>
+                <thead>
+                    <tr>
+                        <th style={{textAlign: "center"}}>Contract Address</th>
+                        <th style={{textAlign: "center"}}>Contract Deployer</th>
+                        <th style={{textAlign: "center"}}>Deployed Block Number</th>
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: Change `Deployed Block Number` to Block #

**Review by passandscore — COMMENTED** [review]
Great start @codyrhoten

File: components/ContractDataTable.tsx (hunk 1 above)
Comment (codyrhoten) [reply components/ContractDataTable.tsx]: This has been completed.

File: components/ContractDataTable.tsx (hunk 2 above)
Comment (codyrhoten) [reply components/ContractDataTable.tsx]: This has been completed.

File: components/ContractDataTable.tsx [hunk 6]
```diff
@@ -1,39 +1,49 @@
...
+        textAlign: "center"
+    },
+
+    tableHeading: {
+        textAlign: "center"
+    },
+}));
+
 export default function ContractDataTable({ contractAddresses }: ContractDataTableProps) {
     // const theme = useMantineTheme();
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: remove dead code

File: components/ContractDataTable.tsx (hunk 3 above)
Comment (passandscore) [reply components/ContractDataTable.tsx]: https://mantine.dev/core/container/

File: components/ContractDataTable.tsx [hunk 7]
```diff
@@ -1,39 +1,49 @@
-import { /* useMantineTheme, */ Text, Table, Badge } from '@mantine/core';
+import { /* useMantineTheme, */ createStyles, Text, Table, Badge } from "@mantine/core";
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: remove un-used imports

File: components/ContractDataTable.tsx (hunk 4 above)
Comment (codyrhoten) [reply components/ContractDataTable.tsx]: This one is completed.

File: components/ContractDataTable.tsx (hunk 3 above)
Comment (codyrhoten) [reply components/ContractDataTable.tsx]: This one is completed.

File: components/ContractDataTable.tsx (hunk 7 above)
Comment (codyrhoten) [reply components/ContractDataTable.tsx]: This is complete.

File: components/ContractDataTable.tsx (hunk 6 above)
Comment (codyrhoten) [reply components/ContractDataTable.tsx]: This is complete.

File: components/ContractDataTable.tsx (hunk 5 above)
Comment (codyrhoten) [reply components/ContractDataTable.tsx]: This is complete.

File: components/ContractDataTable.tsx [hunk 8]
```diff
@@ -38,19 +44,27 @@ export default function ContractDataTable({ contractAddresses }: ContractDataTab
             <Table>
                 <thead>
                     <tr>
-                        <th style={{textAlign: "center"}}>Contract Address</th>
-                        <th style={{textAlign: "center"}}>Contract Deployer</th>
-                        <th style={{textAlign: "center"}}>Deployed Block Number</th>
-                        <th style={{textAlign: "center"}}>Name</th>
-                        <th style={{textAlign: "center"}}>Symbol</th>
-                        <th style={{textAlign: "center"}}>Total Balance</th>
+                        <th style={{ textAlign: "center" }}>Contract Address</th>
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: Short title are better.

Contract Address => Contract

File: components/ContractDataTable.tsx [hunk 9]
```diff
@@ -38,19 +44,27 @@ export default function ContractDataTable({ contractAddresses }: ContractDataTab
...
+                        <th style={{ textAlign: "center" }}>Contract Deployer</th>
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: Short title are better.

Contract Deployer => Deployer

File: components/ContractDataTable.tsx [hunk 10]
```diff
@@ -38,19 +44,27 @@ export default function ContractDataTable({ contractAddresses }: ContractDataTab
...
+                        <th style={{ textAlign: "center" }}>Deployed Block Number</th>
+                        <th style={{ textAlign: "center" }}>Name</th>
+                        <th style={{ textAlign: "center" }}>Symbol</th>
+                        <th style={{ textAlign: "center" }}>Total Balance</th>
```
Comment (passandscore) [inline components/ContractDataTable.tsx]: Total Balance. Is this the number of tokens or nfts on a given vontract or the number or token holders

**Pull request merged**